syntax = "proto3";
package pool;

import "common.proto";


service ModelManager {
    rpc profile_model(ProfileRequest) returns (ProfileResponse);
    rpc divide_model(PartitionRequest) returns (PartitionResponse);
}


message ProfileRequest {
    common.ModelId model_id = 1;
}

message ProfileResponse {
    string model_profile = 1 ;
}

message PartitionRequest {
    common.ModelId model_id = 1;
    string solved_graph = 2;
}

message PartitionResponse {
}